db/sc_config.conf

```diff
@@ -3675,3 +3675,24 @@ SC_MADOGEAR: {
 	}
 	Icon: "SI_MADOGEAR"
 }
+SC_POPECOOKIE: {
+	Icon: "SI_POPECOOKIE"
+}
+SC_VITALIZE_POTION: {
+	Icon: "SI_VITALIZE_POTION"
+}
+SC_SKF_MATK: {
+	Icon: "SI_SKF_MATK"
+}
+SC_SKF_ATK: {
+	Icon: "SI_SKF_ATK"
+}
+SC_SKF_ASPD: {
+	Icon: "SI_SKF_ASPD"
+}
+SC_SKF_CAST: {
+	Icon: "SI_SKF_CAST"
+}
+SC_ALMIGHTY: {
```

Comment:
`NoDeathReset, NoDispelReset, NoClearanceReset` Flags are required and please check the flags for other status in the aegis [EFSTconfig.lua](https://pastebin.com/LuyhJNUv)

Reply:
Done.